sw/device/lib/dif/dif_aon_timer.h

```diff
@@ -119,6 +119,10 @@ dif_aon_timer_result_t dif_aon_timer_init(dif_aon_timer_params_t params,
                                           dif_aon_timer_t *aon);
 
 /** Starts Always-On Timer (wake-up timer).
```

Comment:
Is it worth adding a newline after `/**`while we are here?

Reply:
Ooops, thanks.

Done.